consoletest_finish-gnome-leap15.0-20180416.json

```diff
@@ -0,0 +1,35 @@
+{
+  "tags": [
+    "ENV-DESKTOP-gnome",
+    "ENV-DISTRI-opensuse",
+    "ENV-INSTLANG-en_US",
+    "ENV-VERSION-15.0",

```

Comment:
please avoid version specific needles. This would not match for 15.1 and I think we should try to match it. For SLE we have `ENV-15ORLATER-1`. I think we do not have it for openSUSE, I think, so better just delete this line